README.md

```
@@ -28,6 +28,47 @@ interpret_success(0, "TOKEN\n", marker="TOKEN")   # True  — verified
+⇒ For a §-numbered mandate, read **df-wiki**. For the CI contract, read **df-cicd**.
+
+### Consumer versions — measured 2026-08-31
+
+Adoption of the *code* is complete: across just-akash (18 importers), Blazing-Back (14) and
+akash-github-runner (3) there are **zero local re-definitions** of `qualified_set`,
+`evaluate_provider`, `from_provider_status` or `PreferredSelection`. Nobody has forked the
+logic.
+
+The *version* is one release behind in both consumers:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)